tests/acl/test_acl.py

```diff
@@ -672,6 +720,9 @@ def acl_rules(self, duthosts, localhost, setup, acl_table, populate_vlan_arp_ent
                     self.setup_rules(duthost, acl_table, ip_version)
                     # Give the dut some time for the ACL rules to be applied and LOG message generated
                     time.sleep(30)
+                    wait_until(300, 20, 0, check_create_msg_in_syslog,
+                               duthost, ssh_hdl, LOG_EXPECT_ACL_RULE_CREATE_RE)
```

Comment:
We can probably delete this `sleep(30)` now because this check may return earlier if it finds the create message quicker, resulting in the test running faster.